practice-two/src/App.jsx

```diff
@@ -86,6 +89,26 @@ const App = () => {
     });
   };
 
+  /**
+   * Toggle the products in cart
+   *
+   * @param {string} productID - ID of the product to toggle products cart
+   * @returns {Array} - Updated list of products cart IDs
+   */
+  const toggleProductCart = (productID) => {
```

Comment:
Check naming here. What is the action? I don't understand toggle action

Reply:
Done

Reply:
I don't see any update